_output-itcr.yml

```diff
@@ -8,10 +8,11 @@ bookdown::gitbook:
     toc:
       collapse: section
       before: |
-       <a href="https://www.itcrtraining.org/"><img src="https://raw.githubusercontent.com/jhudsl/DaSL_Course_Template_Bookdown/main/resources/images/logo.png" style="padding-left: 15px; padding-top: 8px;"</a>
+       <a href="https://www.itcrtraining.org/"><img src="assets/ITN_logo.png" style="padding-left: 15px; padding-top: 8px;"</a>
       after: |
-       <p style="text-align:center;"> <a href="https://github.com/jhudsl/DaSL_Course_Template_Bookdown" target="blank" > This content was published with</a> <a href="https://bookdown.org/" target="blank"> bookdown by:</a> </p>
+       <p style="text-align:center;"> <a href="https://github.com/jhudsl/ITCR_Cancer_Research_Leadership" target="blank" > This content was published with</a> <a href="https://bookdown.org/" target="blank"> bookdown by: </a> </p>
```

Comment:
In general this looks good, except that there's some repo specific things that snuck in here that we don't want.

Reply:
fixed this - was this the only place?